### ISSUE TITLE
add specification of txt output format

### ISSUE THREAD

**Issue by eparejatobes** [bg7/BG7#6]
We need something detailing a bit the fields etc

**Comment by pablopareja** [comment]
you mean for the GFF/Fasta/etc formats?

**Comment by eparejatobes** [comment]
Somewhere in the docs it is said that there's a txt output format

**Comment by pablopareja** [comment]
yeah I just checked it and you're right. I had forgotten about the TSV file, (what kind of docs should be added? there already are names for the different columns of the file... :)
